lib/systems/include/PedalsSystem.h

```diff
@@ -166,6 +158,14 @@ class PedalsSystem
     /// @return true or false accordingly
     bool pedal_is_active_(float pedal1ConvertedData, float pedal2ConvertedData, const PedalsParams &params, bool check_mech_activation);
 
+    /// @brief check whether or not SINGLE pedal is active according to input. returns true if pedal is over threshold. removes deadzone.
+    /// @param pedal1ConvertedData the value 0 to 1 of the first pedal without deadzone removed
+    /// @param params the pedal parameters for this specific pedal
+    /// @param check_mech_activation if this is true, function will check percentages against the mechanical activation percentage
+    /// @return true or false accordingly
```

Comment:
What does "true or false accordingly" need? I assume that when check_mech_activation is true, then this returns high when it's above some mechanical brake threshold. When this is false, what does "accordingly" mean?